cfme/services/catalogs/cloud_catalog_item.py

```diff
@@ -157,3 +148,59 @@ def create(self):
         })
         sel.click(template_select_form.add_button)
         flash.assert_success_message('Service Catalog Item "{}" was added'.format(self.name))
+
+
+@navigator.register(CloudCatalogItem, 'All')
+class ItemAll(CFMENavigateStep):
```

Comment:
This navigation destination actually already exists in cfme/services/catalogs/catalog_item.py